virtcontainers/container.go

```diff
@@ -574,7 +574,8 @@ func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) (
 
```

Comment:
Better adding a defer rollback function here for each devices attached.

Reply:
The detach is done on line 577.